src/porepy/models/energy_balance.py

```diff
@@ -679,12 +679,18 @@ def bc_values_enthalpy_flux(self, subdomains: list[pp.Grid]) -> pp.ad.DenseArray
         # Loop over subdomains to collect boundary values
         for sd in subdomains:
             vals = np.zeros(sd.num_faces)
-            # If you know the boundary temperature, do something like: boundary_faces =
-            # self.domain_boundary_sides(sd).all_bf vals[boundary_faces] = (
-            # self.fluid.specific_heat_capacity()
-            # * dirichlet_values
-            # * self.fluid.density()
+            # If you know the boundary temperature, do something like:
```

Comment:
For the reasons I explained in person, I think we should replace the comment by:

If you know the boundary temperature, you might want
to reconstruct the enthalpy flux at the boundary. Using Dirichlet
BCs, the value assigned herein will be multiplied with the Darcy
flux -K \nabla p. Thus, one would normally convert to fluid flux
multiplying by (density/viscosity). Then, if using EnthalpyFromTemperature,
one would compute enthalpy value by multiplying by specific heat
capacity and (boundary temperature - reference temperature).
Note that using non-default constitutive laws in the interior
might require changes to the above recipe.

Reply:
Sounds good to me.

Reply:
Will you implement please, @Yuriyzabegaev, so we can merge this PR?